server/src/routers/companiesRouter.ts

```diff
@@ -5,9 +5,13 @@ import { getCompanies, getCompanyReports } from "../services/companiesServices";
 export const companiesRouter = Router();
 
 companiesRouter.get("/", (_req: Request, res: Response) => {
-  res.json(getCompanies());
+  getCompanies().then((data) => {
+    res.json(data);
+  });
```

Comment:
```suggestion
async (_req: Request, res: Response) => {
  const data = await getCompanies();
  
    res.json(data);
  });
```
też powinno zadziałać :smile: